runtime/gc_stats/CopyForwardStats.hpp

```diff
@@ -70,10 +70,10 @@ class MM_CopyForwardStats : public MM_CopyForwardStatsCore
 	uintptr_t _monitorReferenceCleared; /**< The number of monitor references that have been cleared during marking */
 	uintptr_t _monitorReferenceCandidates; /**< The number of monitor references that have been visited in monitor table during marking */
 
-#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
-	uintptr_t _doubleMappedArrayletsCleared; /**< The number of double mapped arraylets that have been cleared durign marking */
-	uintptr_t _doubleMappedArrayletsCandidates; /**< The number of double mapped arraylets that have been visited during marking */
-#endif /* J9VM_GC_ENABLE_DOUBLE_MAP */
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP) || defined(J9VM_ENV_DATA64)
+	uintptr_t _doubleMappedOrVirtualLargeObjectHeapArrayletsCleared; /**< The number of double mapped or sparse heap allocated arraylets that have been cleared during marking */
+	uintptr_t _doubleMappedOrVirtualLargeObjectHeapArrayletCandidates; /**< The number of double mapped or sparse heap allocated arraylets that have been visited during marking */
```

Comment:
we should report this in VGC

Reply:
But, we should give a more generic external name like 'offheap objects' (that would still make sense if/when we remove support for double mapping). Possibly even renaming the internal names of counters, root scanner tags... (although, since they are not exposed to customer, they can be safely rename at any point of time in future).